### ISSUE TITLE
Add YML examples

### ISSUE THREAD

**Issue by apaskulin** [sensu/sensu-docs#1365]
For all code samples in the Sensu Go docs that support both JSON and YML, apply the language switcher shortcode and add a YML example.

Depends on #1364

**Comment by apaskulin** [comment]
The language toggler is now live! You can add alternative languages to code samples using the following syntax:

```
{{< language-toggle >}}

{{< highlight yml >}}
type: Asset
api_version: core/v2
metadata:
  name: check_script
  namespace: default
spec:
  filters: null
  sha512: 4f926bf4328fbad2b9cac873d117f771914f4b837c9c85584c38ccf55a3ef3c2e8d154812246e5dda4a87450576b2c58ad9ab40c9e2edc31b288d066b195b21b
  url: http://example.com/asset.tar.gz
{{< /highlight >}}

{{< highlight json >}}
{
  "type": "Asset",
  "api_version": "core/v2",
  "metadata": {
    "name": "check_script",
    "namespace": "default"
  },
  "spec": {
    "url": "http://example.com/asset.tar.gz",
    "sha512": "4f926bf4328fbad2b9cac873d117f771914f4b837c9c85584c38ccf55a3ef3c2e8d154812246e5dda4a87450576b2c58ad9ab40c9e2edc31b288d066b195b21b"
  }
}
{{< /highlight >}}

{{< /language-toggle >}}
```

Questions to resolve:
- Within specification tables, we currently use JSON examples. However, these examples aren't valid JSON, so cannot be labelled with `{{< highlight json >}}`. Possible options include: Using only YML examples in spec tables.

**Comment by rgeniesse** [comment]
I have this working for `*.md` files in any number of directories, with any number of JSON blocks, below the directory holding the Go binary.

Example start and output:

Start with:
```
{{< highlight json >}}
{
  "type": "CheckConfig",
  "api_version": "core/v2",
  "metadata": {
    "name": "interval_check",
    "namespace": "default"
  },
  "spec": {
    "command": "check-cpu.sh -w 75 -c 90",
    "subscriptions": ["system"],
    "handlers": ["slack"],
    "interval": 60,
    "publish": true
  }
}
{{< /highlight >}}
```

Output:

```
{{< language-toggle >}}

{{< highlight yml >}}
type: CheckConfig
api_version: core/v2
metadata:
  name: interval_check
  namespace: default
spec:
  command: check-cpu.sh -w 75 -c 90
  handlers:
  - slack
  interval: 60
  publish: true
  subscriptions:
  - system
{{< /highlight >}}

{{< highlight json >}}
{
  "type": "CheckConfig",
  "api_version": "core/v2",
  "metadata": {
    "name": "interval_check",
    "namespace": "default"
  },
  "spec": {
    "command": "check-cpu.sh -w 75 -c 90",
    "subscriptions": ["system"],
    "handlers": ["slack"],
    "interval": 60,
    "publish": true
  }
}
{{< /highlight >}}

{{< /language-toggle >}}
```

Left TODO:

Must have:
- [x] Get the check time in the resulting YAML to be at the top of the data structure, as that is how our JSON ones are and it is a little awkward to not see it at the top for users.
- [x] Test on an actual, full Sensu Go documentation directory.

Nice to have:
- [ ] Get argument ingestion working for the path parent to start the directory traversing from. Currently it will start at the same directory that the executable is in. We could just copy it to the `sensu-go` directory and run it as well.

Remaining questions:
- [X] Does the Sensu Go API accept YML or just JSON? If so, there are a few outstanding questions to answer. If not, we'll just exclude the API directory from conversion. Only supports JSON.
- [X] how to handle (convert to shell tags?): (convert them and need special case for when JSON has no `type` field.
```
{{< highlight json >}}
{
  "deregistration": {
    "handler": "email-handler"
  }
}{{< /highlight >}}
```
- [ ] Will need to update instances where the description above the JSON block calls out what the following is (see RBAC reference line 143).
- [X] Possible issue with `Type` vs `type` or is `Type` invalid? See install/auth.md. `Type` may be valid, but our docs use `type` so conform to that.

**Comment by rgeniesse** [comment]
Results in browser:
<img width="928" alt="Screen Shot 2019-05-27 at 1 39 14 PM" src="https://user-images.githubusercontent.com/19964713/58438686-ee167b80-8084-11e9-99b4-eb6334f387da.png">
